web/src/components/trace/ObservationPreview.tsx

```diff
@@ -33,6 +28,12 @@ export const ObservationPreview = (props: {
   currentObservationId: string;
   traceId: string;
 }) => {
+  const [selectedTab, setSelectedTab] = useState("preview");
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useQueryParam(
    "tab",
    withDefault(StringParam, "preview"),
  );
```

moving this state to a query param makes the view shareable via link

Reply:
the selection of preview vs scores could be stable while clicking through the observations of a trace